### DISCUSSION TITLE
How to return something from notification handler

### DISCUSSION THREAD

**Discussion by ChinmayJoshi-eaton** [hbldh/bleak#451] [Q&A] [answered]
I want to return something from the notification handler. Is there a way to do it?

**Comment by hbldh** [comment]
No, you have to deposit that somewhere: stick it in a global list or, better, send it to a queue.

**Comment by hbldh** [comment, marked as answer]
A rough sketch on how this can be done can be found in the example [`async_callback_with_queue.py`](https://github.com/hbldh/bleak/blob/develop/examples/async_callback_with_queue.py).
